microsoft/testsuites/kvm/kvm_unit_tests.py

```diff
@@ -0,0 +1,55 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import Logger, Node, TestCaseMetadata, TestSuite, TestSuiteMetadata
+from lisa.tools import KvmUnitTests
+
+
+@TestSuiteMetadata(
+    area="virt",
+    category="community",
+    description="""
+    This test suite for executing the community maintained KVM tests at:
+        https://gitlab.com/kvm-unit-tests/kvm-unit-tests
+    """,
+)
+class KvmUnitTestSuite(TestSuite):
+    # These tests take some time to finish executing. The default
+    # timeout of 600 is not sufficient.
+    TIME_OUT = 1200
+
+    @TestCaseMetadata(
+        description="""
+            Runs the kvm-unit-tests suite for Azure VMs.
+        """,
+        priority=3,
+        timeout=TIME_OUT,
+    )
+    def kvm_unit_tests_for_azure_vm(
+        self, log: Logger, node: Node, log_path: Path
+    ) -> None:
+        # TODO: These failures need to be investigated to figure out the exact
+        # cause.
+        expected_failures = [
+            "pmu_lbr",
+            "svm_pause_filter",
+            "vmx",
+            "ept",
+            "debug",
+        ]
+
+        failures = node.tools[KvmUnitTests].run_tests(self.TIME_OUT)
```

Comment:
You might consider moving the timeout as a constant in the tool. Looks like the tool is the bottleneck, so it is better to make it explicit in the tool.

Reply:
`KvmUnitTests` doesn't have any parameter that would modify it's behaviour when called from different tests. The timeout here is inherent to the tool. I would agree with your suggestion if say the tool ran differently based on some parameter for different tests, in which case timeout should be decided by test.

Reply:
Done.